hq_superset/views.py

```diff
@@ -104,6 +107,13 @@ class CCHQApiException(Exception):
     pass
 
 
+def convert_to_array(string_array):
+    array_values = ast.literal_eval(string_array)
+    if array_values and array_values[0] is None:
```

Comment:
I would either
1. give this check a descriptive name
2. extract this to a method which handle these kind of cornercases